stroeer/core/v1/shared.proto

```diff
@@ -6,31 +6,41 @@ option go_package = "github.com/stroeer/tapir/go/stroeer/core/v1;core";
 option java_multiple_files = true;
 option java_package = "de.stroeer.core.v1";
 
-
-// References to get access to the article on different technical platforms like
-// canonical web or AMP.
-message References {
-  // URL path for this article e.g. /section/id_$ID/title.html (required).
-  string path = 1;
-
-  // Canonical URL of the article, may differ if external, e.g. https://www.giga.de/external.html
-  // (required).
-  string canonical = 2;
-
-  // AMP URL of the article.
-  string amp = 3;
+// Represents a single Reference or Link incl. all needed attributes to built
+// an anchor element.
+message Reference {
+  // The type is only needed for filtering multiple links of the same type in
+  // list of references.
+  //
+  // Example entries:
+  // * `title`
+  // * `navigation`
+  // * `sub_navigation`
+  string type = 1;
```

Comment:
FYI: This `type` is not an enum because of total diverse usage. It depends on the context or integration type.

Reply:
@bobaaaaa can you given an example where we would use the type for filtering?

Reply:
Multiple links in a `Stage` or a custom page header.

```ts
// Instead of:
config.subNavigation
config.sponsoringReference
config.titleReference
// We want:
config.references.filter(isSubNavigation)
config.references.filter(isTitle)
config.references.filter(isSponsoring)
```